source/api/plugins/before-spec-api.md

```diff
@@ -0,0 +1,61 @@
+---
+title: before:spec
+---
+
+The `before:spec` event fires before a spec file is run.
+
+The event timing is different depending on if Cypress is run via `cypress run` (run mode) or `cypress open` (interactive mode).
+
+In run mode, the event is fired as soon as possible when a spec is about to be run.
+
+In interactive mode, since there is not a discreet before or after to a spec running, the event is fired once before the browser is launched for a spec.
```

Comment:
```suggestion
- In **run mode**, the event is fired as soon as possible when a spec is about to be run.
- In **interactive mode**, since there is not a discreet before or after to a spec running, the event is fired once before the browser is launched for a spec.
```